faust/agents/agent.py

```diff
@@ -658,7 +658,7 @@ async def _prepare_actor(self, aref: ActorRefT, beacon: NodeT) -> ActorRefT:
         else:
             # agent yields and is an AsyncIterator so we have to consume it.
             coro = self._slurp(aref, aiter(aref))
-        task = asyncio.Task(self._execute_actor(coro, aref), loop=self.loop)
+        task = asyncio.Task(self._execute_actor(coro, aref), loop=self.loop, name=f'{str(aref)}-{self.channel.get_topic_name()}')
```

Comment:
Why are we naming these tasks?

Reply:
For the asyncio loop to print the stack trace

Reply:
hey @patkivikram, I think that this change affected creating agents that are subscribed to multiple topics at the same time. I tested it inside my project and my interpreter is at 3.9. Basically `self.channel.get_topic_name()` raises an exception because there are more then one topics. Can you please revert your changes to the original commit?